src/incognito.ts

```
@@ -63,9 +70,7 @@ export async function detectIncognito(): Promise<{ isPrivate: boolean; browserNa
     }
 
     function assertEvalToString (value: number): boolean {
-	// Using a dummy function instead of eval.toString() to avoid security and performance risks
-  	const dummyFnToStringLen = (function dummyFn(){}).toString().length;
-  	return value === dummyFnToStringLen;
+      return value === eval.toString().length
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)